### PR TITLE
Removed server_url dependency

### PR DESCRIPTION
**Pull request by sdthomas69**
Removed all dependencies on an external server_url configuration (except for the Swagger plugin).  Also fixed some memory settings.

### CODE REVIEW

File: grails-app/conf/application.groovy [hunk 1]
```diff
@@ -37,23 +41,23 @@ environments {
...
         } else {
-            grails.serverURL = "https://api.interactiveoceans.washington.edu/media"
+            server.contextPath = "https://api.interactiveoceans.washington.edu/media"
         }
-        println "serverURL is ${grails.serverURL}"
+        println "server.contextPath is ${server.contextPath}"
     }
     development {
         cavaWpRestUrl = "http://localhost:8888/"
         cavaWpPassword = "iRmQcp@YOCrbRxOHgCT9#4ZT"
```
Comment (lsetiawan) [inline grails-app/conf/application.groovy:52]: Btw, this is just local WP instance password right?

Comment (sdthomas69) [reply grails-app/conf/application.groovy:52]: Yes

**Review by lsetiawan — APPROVED** [review]
Looks good to me, just a small question.

**Comment by lsetiawan** [comment]
Linking to #22

**Pull request merged**